content/applications/marketing/marketing_automation/target_audience.rst

```diff
@@ -1,123 +1,123 @@
 ==================
-Target an audience
+Audience targeting
 ==================
 
-Delivering marketing campaigns to the right audience is paramount when trying to grow a business.
-The Odoo *Marketing Automation* application helps marketers to do just that by providing detailed
-filtering tools, which can be as simple (or as complex) as necessary, to reach the right customers
-at the right time.
+The *Target* and *Filter* section of the campaign form, also referred to as the domain, contains the
+fields used to define the target audience for the campaign's reach (i.e., the unique contact records
+in the database).
 
-Configure target filters
-========================
+The target audience specifies the type of records available for use in the campaign, such as
+*Lead/Opportunity*, *Event Registration*, *Contact*, and more.
 
-When configuring the target filters on a marketing campaign, there are some options that have an
-:guilabel:`> (arrow)` icon beside them. The :guilabel:`> (arrow)` icon signifies that the particular
-filter has more refined parameters within it that can be customized.
+This section can be found on a campaign form by selecting an existing campaign, or :ref:`creating a
+new campaign <marketing_automation/campaigns>` from the :menuselection:`Marketing Automation app -->
+Campaigns` dashboard.
```

Comment:
Don't need lines 9-14; they almost distract away from the small form coverage you're doing in the intro and in the list below.
```suggestion
```